packages/sync-actions/src/product-actions.js

```diff
@@ -106,6 +106,34 @@ export function actionsMapCategories (diff) {
   return removeFromCategoryActions.concat(addToCategoryActions)
 }
 
+export function actionsMapCategoryOrderHints (diff) {
+  const actions = []
+  if (!diff.categoryOrderHints) return actions
+
+  const categoryIds = Object.keys(diff.categoryOrderHints)
+
+  categoryIds.forEach((categoryId) => {
```

Comment:
It is just a preference from my side, but it would have been nicer to see map here.